src/sas/sascalc/dataloader/readers/cansas_reader_HDF5.py

```diff
@@ -200,134 +223,180 @@ def read_children(self, data, parent_list):
+                            u'SASdetector':
                         self.detector.offset.y = float(data_point)
                         self.detector.offset_unit = unit
-                    elif key == u'polar_angle' and self.parent_class == u'SASdetector':
+                    elif key == u'pitch' and self.parent_class == \
+                            u'SASdetector':
                         self.detector.orientation.x = float(data_point)
                         self.detector.orientation_unit = unit
-                    elif key == u'azimuthal_angle' and self.parent_class == u'SASdetector':
+                    elif key == u'roll' and self.parent_class == u'SASdetector':
                         self.detector.orientation.z = float(data_point)
                         self.detector.orientation_unit = unit
-                    elif key == u'beam_center_x' and self.parent_class == u'SASdetector':
+                    elif key == u'yaw' and self.parent_class == u'SASdetector':
+                        self.detector.orientation.y = float(data_point)
+                        self.detector.orientation_unit = unit
+                    elif key == u'beam_center_x' and self.parent_class == \
+                            u'SASdetector':
                         self.detector.beam_center.x = float(data_point)
                         self.detector.beam_center_unit = unit
-                    elif key == u'beam_center_y' and self.parent_class == u'SASdetector':
+                    elif key == u'beam_center_y' and self.parent_class == \
+                            u'SASdetector':
                         self.detector.beam_center.y = float(data_point)
                         self.detector.beam_center_unit = unit
-                    elif key == u'x_pixel_size' and self.parent_class == u'SASdetector':
+                    elif key == u'x_pixel_size' and self.parent_class == \
+                            u'SASdetector':
                         self.detector.pixel_size.x = float(data_point)
                         self.detector.pixel_size_unit = unit
-                    elif key == u'y_pixel_size' and self.parent_class == u'SASdetector':
+                    elif key == u'y_pixel_size' and self.parent_class == \
+                            u'SASdetector':
                         self.detector.pixel_size.y = float(data_point)
                         self.detector.pixel_size_unit = unit
-                    elif key == u'SSD' and self.parent_class == u'SAScollimation':
+                    elif key == u'distance' and self.parent_class == \
+                            u'SAScollimation':
                         self.collimation.length = data_point
                         self.collimation.length_unit = unit
-                    elif key == u'name' and self.parent_class == u'SAScollimation':
+                    elif key == u'name' and self.parent_class == \
+                            u'SAScollimation':
```

Comment:
PEP 8 suggests using parentheses around the condition when it doesn't fit on the line rather than backslash.  This is partly because whitespace after the backslash is a syntax error.